docs/documentation.md

```diff
@@ -570,9 +570,9 @@ You can follow along [on my technical blog](https://ttboj.wordpress.com/).
 
 To report any bugs, please file a ticket at: [https://github.com/purpleidea/mgmt/issues](https://github.com/purpleidea/mgmt/issues).
 
-##Authors
+## Authors
 
-Copyright (C) 2013-2016+ James Shubin and the project contributors
+Copyright © 2013-2016+ James Shubin and the project contributors
```

Comment:
@goberghen Can you point to the spec that mentions needing that space?

The (C) is more accessible for anyone reading it, therefore for this reason I will leave it as is.